src/objects/Hierarchy_Settings__c.object

```diff
@@ -95,29 +95,29 @@
     <fields>
         <fullName>Affl_ProgEnroll_Copy_End_Date__c</fullName>
         <defaultValue>false</defaultValue>
-        <description>Determines if the Affiliation automatically created by creating a Program Enrollment will inherit the Program Enrollment&apos;s Start Date.</description>
+        <description>Indicates whether to set the End Date of the Affiliation created with a Program Enrollment. If enabled, set the End Date to the Program Enrollment&apos;s End Date. If disabled, don't specify the Affiliation&apos;s End Date.</description>
```

Comment:
@lifewithryan I don't know how much it matters, but some of the apostrophes here in this metadata are represented as "`&apos;`" instead of '

Reply:
I went with what I was seeing in the line before replacement. It looked like they were being HTML encoded to ensure consistency (that was my guess) so I just went with it. After spinning them up on an org, I think we are good. HOWEVER -- a very good point here is if we start automatically translating things, I wonder if it will have an effect...